app/views/components/gds/gdsMainTemplate.scala.html

```diff
@@ -0,0 +1,125 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.hmrcfrontend.views.html.helpers.HmrcTrackingConsentSnippet
+@import uk.gov.hmrc.hmrcfrontend.views.html.components.HmrcReportTechnicalIssue
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.reporttechnicalissue.ReportTechnicalIssue
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.language.{En, Cy}
+@import views.BackButton
+@import views.Title
+@import models.requests.AuthenticatedRequest
+@import scala.collection.immutable.Seq
+@import config.AppConfig
+
+@this(
+  govukHeader: GovukHeader,
+  govukLayout: GovukLayout,
+  govukFlexibleLayout: govukFlexibleLayout,
+  govukBackLink: GovukBackLink,
+  siteHeader: components.gds.siteHeader,
+  phaseBanner: components.gds.phaseBanner,
+  hmrcTrackingConsentSnippet: HmrcTrackingConsentSnippet,
+  hmrcReportTechnicalIssue: HmrcReportTechnicalIssue,
+  appConfig: AppConfig
+)
+
+@(
+  title: Title,
+  backButton: Option[BackButton] = None,
+  useCustomContentWidth: Boolean = false
+)(contentBlock: Html)(implicit request: Request[_], messages: Messages)
+
+@useTimeoutDialog = @{ request.isInstanceOf[AuthenticatedRequest[_]]}
+
+@head = {
+    @hmrcTrackingConsentSnippet()
+
+    <link rel="shortcut icon" href='@routes.Assets.versioned("/lib/govuk-frontend/govuk/assets/images/favicon.ico")' type="image/x-icon" />
+    <link rel="shortcut icon" href='@routes.Assets.versioned("lib/accessible-autocomplete/dist/accessible-autocomplete.min.css")' rel="stylesheet" type="text/css" />
+    <meta name="format-detection" content="telephone=no" />
+    <!--[if lte IE 8]><link href='@controllers.routes.Assets.versioned("stylesheets/application-ie-8.css")' rel="stylesheet" type="text/css" /><![endif]-->
+    <!--[if gt IE 8]><!--><link href='@routes.Assets.versioned("stylesheets/customsdecexfrontend-app.css")' media="all" rel="stylesheet" type="text/css" /><!--<![endif]-->
+}
+
+@scripts = {
+    <script src="@routes.Assets.versioned("javascripts/show-hide-content.js")" type="text/javascript"></script>
+
+    @if(useTimeoutDialog) {
+        <script src='@routes.Assets.versioned("lib/hmrc-frontend/hmrc/all.js")'></script>
+        <script src='@routes.Assets.versioned("javascripts/timeoutDialog.js")'> </script>
+        <script>window.HMRCFrontend.initAll();</script>
```

Comment:
Please remove references to `hmrc-frontend` and `initAll`, as we do not import them.

Reply:
Ok, removed